.travis.yml

```diff
@@ -7,9 +7,9 @@ services:
   - postgresql
 
 php:
-  - 5.6
-  - 7.0
+  - 7.2
   - 7.3
```

Comment:
This is not needed. It is also already double covered by postgres and sqlite :)

Reply:
I think I've removed the correct lines. :)

Reply:
You removed the wrong line. 7.2 should stay, 7.3 was marked by me.

Reply:
Min/Max always.

Reply:
Ah ok sorry. Will revert.